pkg/gofr/datasource/pubsub/kafka/mock_interfaces.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by MockGen. DO NOT EDIT.
+//Code generated by MockGen. DO NOT EDIT.
```

Comment:
I don't get how this change was introduced

Do you have a new version of mockgen?